msteams-platform/bots/how-to/create-a-bot-commands-menu.md

```diff
@@ -38,23 +38,23 @@ A prerequisite to create a command menu for your bot is that you must edit an ex
    > [!WARNING]
    > If you have been using App Studio, we recommend that you'd try the Developer Portal to configure, distribute, and manage your Teams apps. App Studio will be deprecated by August 01, 2022.
 
-   :::image type="content" source="conversations/Media/AppStudio.png" alt-text="appstudio-media":::
+   :::image type="content" source="conversations/Media/AppStudio.png" alt-text="The screenshot describes the steps for creating command menu using app studio.":::
 
 2. In **App Studio**, select the **Manifest editor** tab. If you do not have an existing app package, you can create or import an existing app. For more information, see [update C# app package in App Studio](../../get-started/deploy-csharp-app-studio.md).
 
 3. In the left pane of the **Manifest editor** and in the **Capabilities** section, select **Bots**.
 
 4. In the right pane of the **Manifest editor** and in the **Commands** section, select **Add**. The **New Command** screen appears.
 
-   :::image type="content" source="media/AppStudio-CommandMenu-Add.png" alt-text="Select the app package" lightbox="media/AppStudio-CommandMenu-Add.png "border="true":::
+   :::image type="content" source="media/AppStudio-CommandMenu-Add.png" alt-text="The screenshot describes about selecting the app package." lightbox="media/AppStudio-CommandMenu-Add.png ":::
```

Comment:
Not sure if this alt-text correctly describes the screenshot. Please check internally.

Reply:
resolved.